helm/templates/pkgKafkaProvider.yaml

```diff
@@ -0,0 +1,50 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more contributor
+# license agreements; and to You under the Apache License, Version 2.0.
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Values.kafkaprovider.name | quote }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    name: nginx
+spec:
+  replicas: {{ .Values.kafkaprovider.replicaCount }}
+  template:
+    metadata:
+      labels:
+        name: {{ .Values.kafkaprovider.name | quote }}
+    spec:
+      restartPolicy: {{ .Values.kafkaprovider.restartPolicy }}
+      initContainers:
+      # The controller must wait for kafka and couchdb to be ready before it starts
+{{ include "readiness.waitForCouchDB" . | indent 6 }}
+
+      containers:
+      - name: {{ .Values.kafkaprovider.name | quote }}
+        imagePullPolicy: {{ .Values.kafkaprovider.imagePullPolicy | quote }}
+        image: {{ .Values.kafkaprovider.image | quote }}
+        ports:
+        - name: kafkaprovider
+          containerPort: {{ .Values.kafkaprovider.apiPort }}
+        env:
+        - name: "DB_URL"
+          value: "http://$(COUCHDB_SERVICE_HOST):$(COUCHDB_SERVICE_PORT_COUCHDB)"
```

Comment:
to allow external db,
`value: "{{ .Values.db.protocol }}://{{ include "db_host" . }}:{{ .Values.db.port }}"`